packages/myst-cli/src/build/tex/single.ts

```diff
@@ -248,7 +253,12 @@ export async function localArticleToTexTemplated(
     const includeFilenames = results.map((result, ind) => {
       const includeFilename = `${name}-${content[ind]?.slug ?? ind}`;
       const includeFile = path.format({ dir, ext, base: `${includeFilename}${ext}` });
-      writeFileToFolder(includeFile, result.value);
+      let part = '';
+      const { title, content_includes_title } = content[ind]?.frontmatter ?? {};
+      if (title && !content_includes_title) {
+        part = `\\section{${title}}\n\n`;
```

Comment:
I'm not sweating too much about duplicated code here - it is going to change almost immediately with (1) loading from a toc, where this `section`/`chapter` level is variable and (2) sub-article templating.